sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -569,46 +569,51 @@ class SessionCatalog(
   /**
    * Rename a table.
    *
-   * If a database is specified in `oldName`, this will rename the table in that database.
+   * If the database specified in `newName` is different from the one specified in `oldName`,
+   * It will result in moving table across databases.
+   *
    * If no database is specified, this will first attempt to rename a temporary table with
-   * the same name, then, if that does not exist, rename the table in the current database.
+   * the same name, then, if that does not exist, current database will be used for locating
+   * `oldName` or `newName`.
```

Comment:
This is wrong, right? This is for `temp view`

Reply:
So should I make the comment here unchanged?